cmd/auth-handler.go

```diff
@@ -180,24 +193,32 @@ func getClaimsFromToken(r *http.Request, cred auth.Credentials) (map[string]inte
 		}
 		return nil, errAuthentication
 	}
-	claims := make(map[string]interface{})
+	p := &jwtgo.Parser{}
+	jtoken, err := p.ParseWithClaims(token, jwtgo.MapClaims(claims), stsTokenCallback)
```

Comment:
The `stsTokenCallback` function does not check if the accessKey in the claims is the same as the one that is in the globalServerConfig - this seems like an incomplete check and possible security issue?.